projects/go-lib/src/lib/components/go-radio/go-radio-group.component.html

```diff
@@ -1,11 +1,11 @@
 <fieldset
-  class="go-form__fieldset go-form__fieldset--no-margin"
   [disabled]="control.disabled"
-  [ngClass]="{ 'go-form__fieldset--error': control?.errors?.length, 'go-form__fieldset--dark': theme === 'dark' }"
+  [ngClass]="{ 'go-form__fieldset--error': control?.errors?.length, 'go-form__fieldset--dark': theme === 'dark', 'go-form__fieldset--disable': enableFieldset, 'go-form__fieldset go-form__fieldset--no-margin': enableFieldset }"
```

Comment:
> Should `'go-form__fieldset go-form__fieldset--no-margin': enableFieldset` be just `'go-form__fieldset--no-margin': enableFieldset`?

After taking a look, it can be simplified further. The `<fieldset>` now has this `ngClass`:
```html
[ngClass]="{
    'go-form__fieldset--error': control?.errors?.length,
    'go-form__fieldset--dark': theme === 'dark',
    'go-form__fieldset': enableFieldset
  }"
```

I removed the `margin-bottom` from `go-form__fieldset` because that would be achieved through using the grid properly which allowed me to remove the `go-form__fieldset--no-margin` on the `ngClass` altogether.

> Also, is `'go-form__fieldset--disable': enableFieldset` correct? Why is the `--disable` class being added to the fieldset when `enableFieldset` is `true`? Seems like it should be `'go-form__fieldset--disable': !enableFieldset`, semantically speaking.

Yeah, we don't even need this line. We apply styles based on if the `fieldset` has the `disabled` attribute set via the cascade. So this class didn't really do anything.

> Lastly, should we break this up into multiple lines?

Sure, I cleaned it up a bit.